ml4h20.bib

```diff
@@ -2,20 +2,20 @@ @Proceedings{ML4H-2019
     booktitle = {Proceedings of the Machine Learning for Health NeurIPS Workshop},
     name = {Machine Learning for Health Workshop},
     shortname = {ML4H},
-    editor = {Adrian V. Dalca and Matthew Mcdermott and Emily Alsentzer and Sam Finlayson and Michael Oberst and Fabian Falck and Brett Beaulieu-Jones},
+    editor = {Adrian V. Dalca and Matthew B.A. McDermott and Emily Alsentzer and Samuel G. Finlayson and Michael Oberst and Fabian Falck and Brett Beaulieu-Jones},
     volume = {1},
     year = {2020},
     start = {2019-12-13},
     end = {2019-12-13},
     url = {https://ml4health.github.io/2019},
     location = {Vancouver, British Columbia, Canada},
-	published = {2020-04-30}
+    published = {2020-04-30}
 }
 
 @InProceedings{dalca20,
-author = {Dalca, Adrian V and Mcdermott, Matthew and Alsentzer, Emily and Finlayson, Sam and Oberst, Michael and Falck, Fabian and Chivers, Corey and Beam, Andrew and Naumann, Tristan and Beaulieu-jones, Brett},
+author = {Dalca, Adrian V and McDermott, Matthew B.A. and Alsentzer, Emily and Finlayson, Samuel G. and Oberst, Michael and Falck, Fabian and Chivers, Corey and Beam, Andrew and Naumann, Tristan and Beaulieu-jones, Brett},
 pages = {1-9},
-title = {{Machine Learning for Health ( ML4H ) 2019 : What Makes Machine Learning in Medicine Different ?}}
+title = {{Machine Learning for Health ( ML4H ) 2019 : What Makes Machine Learning in Medicine Different?}}
```

Comment:
Similarly, these changes need to be made in this file (not the .bib file): https://github.com/mlresearch/v116/edit/gh-pages/_posts/2020-04-30-dalca20a.md